### PR TITLE
Fill in details for CVE-2020-8561.

### PR DESCRIPTION
**Pull request by cjcullen**

### CODE REVIEW

File: validate-k8s.sh [hunk 1]
```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```
Comment (tallclair) [inline validate-k8s.sh:1]: Please revert.

Comment (cjcullen) [reply validate-k8s.sh]: Done

File: 2020/8xxx/CVE-2020-8561.json [hunk 2]
```diff
@@ -1,18 +1,119 @@
...
+                                "product_name": "Kubernetes",
+                                "version": {
+                                    "version_data": [
+                                        {
+                                            "version_affected": "<=",
+                                            "version_value": "1.20.1"
+                                        }, 
+                                        {
+                                            "version_affected": ">?",
+                                            "version_value": "1.20.11"
```
Comment (tallclair) [inline 2020/8xxx/CVE-2020-8561.json:25]: typo?
```suggestion
                                            "version_value": "1.20.1"
```

Comment (cjcullen) [reply 2020/8xxx/CVE-2020-8561.json:25]: well, the other one was a typo. Fixed.

**Review by tallclair — APPROVED** [review]
lgtm

**Pull request merged**